File: Patterns/5. Right Triangle Patterns (Bottom-Left)/50. Right Triangle BL Continuous Numbers/blcn.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int n,num=1;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        for(int j=i; j<=n; j++){ // for(int j=1; j<=(n-i+1); j++)
            cout<<setw(3)<<num ; 
            num++;
        }
        cout<<'\n';
    }
}

/*

Use (i+j<=n+1) for optimized code & for n matrix 

Output

  1  2  3  4  5
  6  7  8  9
 10 11 12
 13 14
 15   

*/

File: Patterns/05. Right Triangle Patterns (Bottom-Left)/50. Right Triangle BL Continuous Numbers/blcn.py
n = int(input("Enter a Number : "))
num=1

for i in range(1,n+1):
    for j in range(i,n+1): # for j in range(1,n-i+2):
        print(f"{num:3}", end='') 
        num+=1
    print()

'''

Use (i+j<=n+1) for optimized code & for n matrix 

Output:

  1  2  3  4  5
  6  7  8  9
 10 11 12
 13 14
 15 
 
'''